src/views/Admin.vue

```diff
@@ -293,8 +309,10 @@ export default {
       firebase
         .auth()
         .signOut()
-        .then(function() {
+        .then(() => {
```

Comment:
This is a funny one, where `function() { ... }` doesn't bind the containing `this`. I almost always use `=>` because of this. Ahhhh javascript`